container/container.go

```diff
@@ -456,6 +456,11 @@ func (x Domain) Zone() string {
 	return "container"
 }
 
+// EncodeToString domain into string.
+func (x Domain) EncodeToString() string {
```

Comment:
Is the string format specified somewhere? I realize it is correct, but still. If there is no document with it, I suggest to form the string in `NNS.ResolveContainerName` method.